### PR TITLE
Introduce the oracle db script

### LINKED ISSUE #205: Oracle DB support
### Description

We need to provide Oracle DB scripts for message and broker semantic persistence.

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
Introduce the database script for Oracle. Resolves #205 

## Goals
Allow using the broker with Oracle databases.

## Approach
Introduce the database script for Oracle.

## User stories
N/A

## Release note
N/A

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
N/A
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
N/A

## Related PRs
#106 #191 

## Migrations (if applicable)
N/A

## Test environment
Oracle 11g
 
## Learning
N/A

### CODE REVIEW

File: modules/launcher/src/main/resources/dbscripts/oracle-mb.sql [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+-- WSO2 Message Broker Oracle Database schema --
+
+-- Start of Message Store Tables --
+/
+CREATE TABLE MB_EXCHANGE (
```
Comment (a5anka) [inline modules/launcher/src/main/resources/dbscripts/oracle-mb.sql]: Can we add the "if not exists" check?

Comment (MaryamZi) [reply modules/launcher/src/main/resources/dbscripts/oracle-mb.sql]: There doesn't seem to be a straightforward/simple way to do an "if exists" check for table creation with Oracle. We would have to drop the tables if they exist separately (keeping this script simple), prior to creating the tables?

Comment (a5anka) [reply modules/launcher/src/main/resources/dbscripts/oracle-mb.sql]: 👍 Let's keep the script as it is then. Shall we mention that we need to drop old tables in the doc?

Comment (MaryamZi) [reply modules/launcher/src/main/resources/dbscripts/oracle-mb.sql]: Created #233 to track introducing relevant sections in the database setup guide. Will include dropping old tables for Oracle.

**Pull request merged**